html/cross-origin-embedder-policy/require-corp.https.html

```diff
@@ -0,0 +1,130 @@
+<!doctype html>
+<meta name="timeout" content="long">
+<title>Cross-Origin-Embedder-Policy header and nested navigable resource without such header</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src="/common/get-host-info.sub.js"></script>
+<script src="/common/utils.js"></script> <!-- Use token() to allow running tests in parallel -->
+<div id=log></div>
+<script>
+async_test(t => {
+  const frame = document.createElement("iframe");
+  t.step_timeout(() => {
+    // Make sure the iframe didn't load.
+    assert_equals(frame.contentDocument, null);
+    t.done();
+  }, 500);
+  frame.src = "/common/blank.html";
+  document.body.append(frame);
+  assert_equals(frame.contentDocument.body.localName, "body");
+}, `"require-corp" top-level: navigating a frame to "none" should fail`);
```

Comment:
Per 3.2.1.6 in https://mikewest.github.io/corpp/#corp-check
CORP will be "same-origin". We might set a cross-origin navigation in iframe

Reply:
Okay, the test is for COEP: "require-corp" to COEP: "none"